tests/e2e/coach-runner-relationships.spec.ts

```
@@ -0,0 +1,350 @@
+/**
+ * Coach-Runner Relationship E2E Tests
+ *
+ * Tests the complete coach-runner relationship management flow,
+ * ensuring proper relationship creation, management, and state updates.
+ */
+import { expect, test } from '@playwright/test'
+
+import { TEST_USERS } from '../utils/test-helpers'
+import { navigateToPage, signIn, waitForNavigation } from '../utils/wait-helpers'
```

ESLint

[error] 'waitForNavigation' is defined but never used.

(@typescript-eslint/no-unused-vars)